src/Command/DocsCommand.php

```diff
@@ -26,7 +31,82 @@ protected function configure() {
    * @return int
    */
   protected function execute(InputInterface $input, OutputInterface $output) {
-    $this->localMachineHelper->startBrowser('https://docs.acquia.com/acquia-cli/');
+    $acquia_products = [
+      'Acquia CLI' => [
+        'url' => 'acquia-cli',
+        'alias' => ['cli', 'acli'],
+      ],
+      'Acquia CMS' => [
+        'url' => 'acquia-cms',
+        'alias' => ['acquia_cms', 'acms'],
+      ],
+      'Code Studio' => [
+        'url' => 'code-studio',
+        'alias' => ['code_studio', 'codestudio'],
```

Comment:
Also referred to as `cs`. Might be a good idea to add it too.

Reply:
updated!